.github/codecov.yml

```diff
@@ -1,7 +1,17 @@
+# https://docs.codecov.io/docs/commit-status
+coverage:
+  status:
+    project:
+      default:
+        informational: true
+    patch: off
```

Comment:
Annotations are a bit overkill, so I've disabled them with `patch: off`, and I don't think we care too much about the coverage of a PR's diff.

![image](https://user-images.githubusercontent.com/567105/118982973-da503800-b973-11eb-8957-825655b79d49.png)

![image](https://user-images.githubusercontent.com/567105/118982990-de7c5580-b973-11eb-8c03-e574743d4c69.png)

![image](https://user-images.githubusercontent.com/567105/118982777-a6751280-b973-11eb-9be4-f50645cc0815.png)